apps/extension/src/utils/taskFileReader.ts

```
@@ -0,0 +1,214 @@
+ * Reads tasks.json file directly and extracts implementation details and test strategy
+ * @param taskId - The ID of the task to read (e.g., "1" or "1.2" for subtasks)
+ * @param tagName - The tag/context name (defaults to "master")
+ * @returns TaskFileData with details and testStrategy fields
+ */
+export async function readTaskFileData(
+	taskId: string,
+	tagName: string = 'master'
+): Promise<TaskFileData> {
+	try {
+		// Check if we're in a VS Code webview context
+		if (typeof window !== 'undefined' && (window as any).vscode) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)